gum/backend-linux/gumandroid.c

```diff
@@ -321,8 +321,9 @@ struct _GumFunctionSignature
 };
 
 static const GumModuleDetails * gum_try_init_linker_details (void);
+static gchar * gum_find_linker_path (void);
 static gboolean gum_try_parse_linker_proc_maps_line (const gchar * line,
-    GumModuleDetails * module, GumMemoryRange * range);
+    const gchar * linker_path, GumModuleDetails * module, GumMemoryRange * range);
```

Comment:
```suggestion
    const gchar * linker_path, GumModuleDetails * module,
    GumMemoryRange * range);
```

(Overflows 80 cols by two spaces, so we need to break it.)

Reply:
Hah, you should post your linter config :grin: